.gitignore

```diff
@@ -13,12 +13,13 @@ rerun.txt
 pickle-email-*.html
 
 config/initializers/secret_token.rb
-config/secrets.yml
+config/local_env.yml
 .rbenv-vars
```

Comment:
Back when I was reading up on how to hide Devise secret key as environment var, I put the actual secret key in this file instead of secret.yml so I gitignore it.